<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OfertasForm</class>
 <widget class="QWidget" name="OfertasForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 170, 255);</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>691</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lblTitulo_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>661</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Título</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblProfesional">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>621</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Profesional</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblEdades">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>641</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Para las siguientes edades</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblContenido">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>631</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string>Contenido</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnActualizar">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>Actualizar</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnNoInteresa">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>530</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>No me interesa</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnInteresa">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>530</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);</string>
   </property>
   <property name="text">
    <string>Me interesa</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitulo">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>691</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Posts del día</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCrearPost">
   <property name="geometry">
    <rect>
     <x>789</x>
     <y>90</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 127);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Crear un post</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMisPost">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>130</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 127);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Mis post</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSalir">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>210</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 85, 0);</string>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnKarma">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>170</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Rey del karma</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
